profile/README.md

```diff
@@ -0,0 +1,15 @@
+## Teleport
+
+Welcome to the @graviational Github Org, home of [Teleport](https://goteleport.com/). Teleport is the easiest, most secure way to access infrastructure. The open-source Teleport Access Plane consolidates connectivity, authentication, authorization, and audit into a single platform. By consolidating all aspects of access for engineers and the software they write, Teleport reduces attack surface area, cuts operational overhead, easily enforces compliance, and improves productivity.
+
+## Open-Core at Teleport
+Teleport is an [open-core product](https://en.wikipedia.org/wiki/Open-core_model), with all [core features](https://goteleport.com/features/) being available in the community edition of Teleport. The engineering team develops and designs in the open. With everything from Design Documents, [RFDs](https://github.com/gravitational/teleport/tree/master/rfd), [Core Code](https://github.com/gravitational/teleport/) to [Test Plans](https://github.com/gravitational/teleport/issues?q=is%3Aclosed+Test+Plan) being created in the open. 
+
+## Join the Community
+The [Teleport community](https://goteleport.com/community/) is comprised of people using Teleport at work, home or any situation that needs to provide secure infrastructure access. We maintain a community [Slack](https://goteleport.com/slack/), answer questions in our [Github Discussions board](https://github.com/gravitational/teleport/discussions). 
+
+## Work with Us 
+Teleport is a remote-first technology company with satellite offices in Oakland and Seattle. View our available career opportunities [on our careers site](https://goteleport.com/careers/).
+
+## History of @graviational Github Org
+Teleport started it's life as Gravitational. What we built at Gravitational is Gravity and Teleport. The Gravity project was meant to be the “runtime” for this computer, while the goal of Teleport was to provide an easy way to access it. In November of 2020, [Gravitational re-branded as Teleport](https://goteleport.com/blog/gravitational-is-teleport/) and since then we've continued to execute Teleports mission to enable engineers to quickly access any resource anywhere.
```

Comment:
```suggestion
Teleport started its life as Gravitational. What we built at Gravitational was Gravity and Teleport. The Gravity project was meant to be the “runtime” for this computer, while the goal of Teleport was to provide an easy way to access it. In November of 2020, [Gravitational re-branded as Teleport](https://goteleport.com/blog/gravitational-is-teleport/), and since then we've continued to execute Teleport's mission to enable engineers to quickly access any resource anywhere.
```